plugin.yaml

```diff
@@ -1161,12 +1359,12 @@ relationships:
             group_self_url:
               default: { get_attribute: [TARGET, selfLink] }
 
-  cloudify.gcp.relationships.contained_in_compute:
-    derived_from: cloudify.relationships.contained_in
-    source_interfaces:
-      cloudify.interfaces.relationship_lifecycle:
-        preconfigure:
-          implementation: gcp_plugin.cloudify_gcp.compute.instance.contained_in
+#  cloudify.gcp.relationships.contained_in_compute:
+#    derived_from: cloudify.relationships.contained_in
+#    source_interfaces:
+#      cloudify.interfaces.relationship_lifecycle:
+#        preconfigure:
+#          implementation: gcp_plugin.cloudify_gcp.compute.instance.contained_in
```

Comment:
Please remove rather than commenting obsolete code.
We retain history in git so there's no need to hang onto commented-out code like this

Reply:
@funkyHat  yeah I will remove this, I'm not done yet with this code :)

Reply:
@funkyHat This is most likely while I was debugging and testing , will fix this